app/src/org/commcare/heartbeat/HeartbeatRequester.java

```diff
@@ -116,10 +118,16 @@ protected void requestHeartbeat() {
         params.put(CC_VERSION, ReportingUtils.getCommCareVersionString());
         params.put(QUARANTINED_FORMS_PARAM, "" + StorageUtils.getNumQuarantinedForms());
         params.put(UNSENT_FORMS_PARAM, "" + StorageUtils.getNumUnsentForms());
-        params.put(LAST_SYNC_TIME_PARAM, new Date(SyncDetailCalculations.getLastSyncTime()).toString());
+        params.put(LAST_SYNC_TIME_PARAM, getISO8601FormattedLastSyncTime());
         return params;
     }
 
+    private static String getISO8601FormattedLastSyncTime() {
+        DateFormat df = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm'Z'");
+        df.setTimeZone(TimeZone.getTimeZone("UTC"));
+        return df.format(SyncDetailCalculations.getLastSyncTime());
```

Comment:
is `getLastSyncTime()` guaranteed to return non-null?

Reply:
`getLastSyncTime()` returns a `long` so it is guaranteed to be non-null, but you're probably right that I should be sending something other than 1/1/1970 if we haven't recorded one yet (although theoretically that should never happen anymore).

@snopoke Is it ok for me to send something like "Never" if we don't have a last sync time recorded?